_generated/MavenAuthenticateV0_Node20/mavenauth.ts

```diff
@@ -54,6 +60,33 @@ async function run(): Promise<void> {
             tl.debug(tl.loc("Info_CreatingSettingsXml", userSettingsXmlPath));
             tl.setVariable('FIRST_RUN_SETTINGS_XML_EXISTS_PATH', userSettingsXmlPath);
         }
+        
+        if (feedUrl && entraWifServiceConnectionName) {
```

Comment:
Wrap all the logic include variable in wif

Reply:
Updated this!